src/components/splashScreen/TitleBadge.tsx

```diff
@@ -1,14 +1,36 @@
+import { useState } from "react";
 import styles from "./TitleBadge.module.css";
 
 type TitleBadgeProps = {
   isPaniniOrdered: boolean;
+  handleTransition: () => void;
 };
 
 export default function TitleBadge(props: TitleBadgeProps) {
+  const [isButtonAvailable, setIsButtonAvailable] = useState(true);
+  const handleClick = () => {
+    setIsButtonAvailable(false);
+    setTimeout(() => {
+      setIsButtonAvailable(true);
+    }, 4000);
+  };
+  const buttonStyle = {
+    cursor: "initial",
+  };
   return (
-    <div className={styles.titleBadge}>
+    <div className={`${styles.titleBadge} ${!isButtonAvailable && styles.titleBadgeEscape}`}>
       <h1 className={styles.textContent}>Panini {props.isPaniniOrdered ? "ordered" : "Creator"}</h1>
-      <button className={styles.button}>{props.isPaniniOrdered ? "start again" : "begin"}</button>
+      <button
+        className={styles.button}
+        onClick={() => {
```

Comment:
**nitpick:** Writing inline handlers is considered a bad practice - you can put this `props.handleTransition` into `handleClick`, so you can do:

```tsx
 onClick={handleClick}
```